pipelines/visitQAEllipPlots.yaml

```diff
@@ -270,6 +270,29 @@ tasks:
         from lsst.analysis.drp.calcFunctors import CalcShapeSize
         from lsst.pipe.tasks.dataFrameActions import SubtractColumns
 
+  plot_shapeSizeResidsDiff_scatter_visit:
+    class: lsst.analysis.drp.scatterPlotVisit.ScatterPlotVisitTask
+    config:
+      axisActions.yAction: FractionalResidualColumns
+      axisActions.yAction.colA: CalcShapeSize
+      axisActions.yAction.colA.colXx: "ixx"
+      axisActions.yAction.colA.colYy: "iyy"
+      axisActions.yAction.colB: CalcShapeSize
+      axisActions.yAction.colB.colXx: "ixxPSF"
+      axisActions.yAction.colB.colYy: "iyyPSF"
+      connections.plotName: shapeFractionDiff_sizePSF_scatter
+      selectorActions.flagSelector: VisitPlotFlagSelector
+      selectorActions.SnSelector: SnSelector
+      selectorActions.SnSelector.bands: [""]
+      selectorActions.SnSelector.fluxType: "psfFlux"
+      selectorActions.SnSelector.threshold: 10
+      axisLabels: {"x": "PSF Magnitude (mag)", "mag": "PSF Magnitude (mag)",
```

Comment:
Is the `mag` entry in this dict required, as it seems to be a repeat of `x`?

Reply:
This follows from `axisLabels` in other plots. I don't understand why `mag` and `x` entries are required when the string appears exactly once in the plot. This will be addressed in DM-33971 as well.

Reply:
On second thoughts, I think `mag` entry is associated with the stats that appears on the bottom right corner of the plot.